ads/google/correlator.js

```diff
@@ -24,6 +24,8 @@ export function makeCorrelator(clientId, pageViewId) {
   if (clientId) {
     return pageViewIdNumeric + (clientId.replace(/\D/g, '') % 1e6) * 1e6;
   } else {
-    return pageViewIdNumeric;
+    // In this case, pageViewIdNumeric is only 4 digits => too low entropy
+    // to be useful as a page correlator.  So synthesize one from scratch.
+    return Math.floor(Math.pow(2, 52) * Math.random());
```

Comment:
I think I got 52 from the size of JS numbers.  (Floating point theoretical precision or something.)  But I admit that it might not reflect the real resolution of the PRNG.  But it's okay to be bigger -- we just don't want it to be smaller.  Don't want to unnecessarily lose precision.

Happy to hardcode the constant.  Will do that shortly.